src/components/view/Contact/ContactProfileCard/index.js

```diff
@@ -0,0 +1,63 @@
+
+import React, { Component } from 'react';
+
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+class ContactProfileCard extends React.Component {
+
+  render() {
+    const { contact } = this.props;
+
+    const infoKeyStyle = {
+      color: '#90A4AE',
+      'padding-right': '.5rem'
+    }
+
+    return (
+      <div>
+        <Card initiallyExpanded={true} >
+          <CardHeader
+              title='Profile'
+              titleColor={this.props.titleColor}
+              actAsExpander={true}
+              showExpandableButton={true}
+          />
+          <CardText expandable={true}>
+            <div className="row">
+              <div className="col-xs-6">
```

Comment:
Could you try experimenting with the col sizing across different page widths, just to test out smallest size vs desktop? It's not formally required, but I bet it will be a quick check and will handle more mobile screens better.

Reply:
it looks good to me. on a phone it will be 2 columns